src/app.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const app = express();
+
+const artistRouter = require('./routes/artist');
+const albumRouter = require('./routes/album');
+
+app.use(express.json());
+
+
+app.use("/artists", artistRouter);
+app.use("/artists", albumRouter);
+app.use("/albums", albumRouter);
```

Comment:
Replace double quotes with single as that's what's used throughout the project
```suggestion
app.use('/artists', artistRouter);
app.use('/artists', albumRouter);
app.use('/albums', albumRouter);
```